.github/workflows/release.yaml

```diff
@@ -0,0 +1,249 @@
+# This workflow is used to publish SV2 crates to cratesio as well as releasing binaries of the roles
+# (translator, pool, mining_proxy) to github. The github releases will fail if there are no tags available
+# so the `autoversion` workflow must be ran and the resulting PR must be merged at least once before this will 
+# release the binaries to github. 
+
+# If the binary releases do fail to due to not having tags, force run the `autoversion` workflow
+# on the main branch and merge the resulting PR to create the tags and move them to the main branch.
+
+# This should be disabled for forks
+
+name: Release
+
+on: 
+  # Manually run by going to "Actions/Release" in Github and running the workflow
+  workflow_dispatch:
+
+jobs:
+  # Cannot run until we have necessary keys to publish crates to crates.io.
+  # This should not be run on forks since removing the `--no-changelog` flag in the "Commit version updates"
+  # step will create all changelog files with respect to the fork this was executed in, so each changlog file
+  # will log the forks history of the crate instead of the main repos history
+  crates_publish:
+    name: Release
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: 
+          - ubuntu-latest
+        include:
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-musl
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: 
+        run: echo TEST
+      
+      # - name: Install cargo-smart-release
+      #   run: cargo install cargo-smart-release
+
+      # # publish and create changelogs
+      # - name: Commit version updates
+      #   run: sh ./sv2-publish.sh -e -u --no-push --no-changelog
+      
+      # # Push changes to bot/versioning branch
+      # - name: Push version updates to main
+      #   run: git push -f --no-verify origin main:bot/versioning
+
+      # # PR updates
+      # - name: Create pull request
+      #   uses: peter-evans/create-pull-request@v4
+      #   with:
+      #     base: main
+      #     branch: bot/versioning
+
+  github_release_translator:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # This defines what environments we will build on
+        include:
+          - os: ubuntu-latest
+            artifact_name: translator_sv2
+            asset_name: translator_sv2-linux-amd64
+          # - os: windows-latest
```

Comment:
Im assuming this is commented out because it doesnt work on windows?

Reply:
Yep. I will make a ticket once this gets merged